features/step_definitions/your_contributions_steps.rb

```diff
@@ -0,0 +1,53 @@
+Given(/^I am on the YourDetailsPage$/) do
+  @your_details_page = UI::YourDetails.new
+  @your_details_page.load
+end
+
+When(/^I enter my age as 35$/) do
+  @your_details_page.age.set 35
+end
+
+When(/^I enter my gender as female$/) do
+  @your_details_page.genders.select('Female')
+end
+
+When(/^my salary per year is equal to or less than the upper earnings threshold of £45,000$/) do
+  @your_details_page.salary.set 35000
```

Comment:
I'm ok for this to be merged for now as we haven't established our full code standards yet, but it would be good to follow one convention of either using `()` or not. Was it a conscious decision not to wrap `35000` in `()`? 
Other lines are following the `()` convention:
`.select('per Year')`
`have_content(39124)`